build

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env bash
+
+# build - build Jellyfin binaries or packages
+
+set -o errexit
+set -o pipefail
+
+# The list of possible package actions (except 'clean')
+declare -a actions=( 'build' 'package' 'sign' 'publish' )
+
+# The list of possible platforms, based on directories under 'deployment/'
+declare -a platforms=( $(
+    find deployment/ -maxdepth 1 -mindepth 1 -type d | sed 's/deployment\///' | sort
+) )
+
+# The list of standard dependencies required by all build scripts; individual
+# action scripts may specify their own dependencies
+declare -a dependencies
+dependencies=( 'tar' 'zip' 'dotnet' )
+
+usage() {
+    echo -e "build - build Jellyfin binaries or packages"
+    echo -e ""
+    echo -e "Usage:"
+    echo -e " $ build --list-platforms"
+    echo -e " $ build --list-actions <platform>"
+    echo -e " $ build [-b/--web-branch <web_branch>] <platform> <action>"
+    echo -e ""
+    echo -e "The web_branch defaults to 'master'."
+    echo -e "To build all platforms, use 'all'."
+    echo -e "To perform all build actions, use 'all'."
+    echo -e "Build output files are collected at '../jellyfin-build/<platform>'."
+}
+
+if [[ -z $1 ]]; then
+    usage
+    exit 0
```

Comment:
Shouldn't exit 0 here as calling the program without an arg is invalid. Only `-h` and `--help` should show usage and exit 0.

Reply:
My intent is that argless is the same as -h or --help, but I will make them explicit too.